$MirOS: ports/www/dillo/v2/patches/patch-src_paths_cc,v 1.1 2009/08/23 15:27:59 bsiegert Exp $
--- src/paths.cc.orig	Wed Jun 24 18:17:12 2009
+++ src/paths.cc	Mon Aug 24 16:29:45 2009
@@ -25,7 +25,7 @@
 static char* oldWorkingDir = NULL;
 
 /*
- * Changes current working directory to /tmp and creates ~/.dillo
+ * Changes current working directory to /tmp and creates ~/.etc/dillo
  * if not exists.
  */
 void Paths::init(void)
@@ -42,7 +42,7 @@ void Paths::init(void)
           dStrerror(errno));
    }
 
-   path = dStrconcat(dGethomedir(), "/.dillo", NULL);
+   path = dStrconcat(dGethomedir(), "/.etc/dillo", NULL);
    if (stat(path, &st) == -1) {
       if (errno == ENOENT) {
          MSG("paths: creating directory %s.\n", path);
@@ -80,7 +80,7 @@ void Paths::free(void)
 FILE *Paths::getPrefsFP(const char *rcFile)
 {
    FILE *fp;
-   char *path = dStrconcat(dGethomedir(), "/.dillo/", rcFile, NULL);
+   char *path = dStrconcat(dGethomedir(), "/.etc/dillo/", rcFile, NULL);
 
    if (!(fp = fopen(path, "r"))) {
       MSG("paths: Cannot open file '%s'\n", path);
